common/shlibs

```diff
@@ -3971,3 +3971,4 @@ libwayland-cursor++.so.0 libwaylandpp-0.2.8_1
 libwayland-egl++.so.0 libwaylandpp-0.2.8_1
 libwayland-client-extra++.so.0 libwaylandpp-0.2.8_1
 libspdlog.so.1 libspdlog-1.8.2_2
+libsane-airscan.so.1 sane-airscan-0.99.24_1
```

Comment:
This isn't necessary, because nothing links against it. The warning about the library not being in `common/shlibs` is harmless.

Reply:
makes sense :sparkles: